common/core/web/keyboard-processor/test.sh

```diff
@@ -0,0 +1,65 @@
+#! /bin/bash
+
+# We should work within the script's directory, not the one we were called in.
+cd $(dirname "$BASH_SOURCE")
+
+# Include useful testing resource functions
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../../../../resources/build/build-utils.sh"
```

Comment:
Can you use the boilerplate version of this include (including the comments) so that it's consistent with other uses:

```suggestion
# Include useful testing resource functions

## START STANDARD BUILD SCRIPT INCLUDE
# adjust relative paths as necessary
THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
. "$(dirname "$THIS_SCRIPT")/../../../../resources/build/build-utils.sh"
## END STANDARD BUILD SCRIPT INCLUDE
```